members/index.html

```diff
@@ -1,24 +0,0 @@
----
```

Comment:
I'm also not a Git guru yet, so I've never rebased in a repository. Should I just google it, or do you want to explain a little bit more about exactly that entails?